WalletWasabi/WabiSabi/Backend/Rounds/Round.cs

```diff
@@ -31,7 +31,7 @@ public enum EndRoundState
 
 public class Round
 {
-	public Round(RoundParameters parameters, WasabiRandom random)
+	public Round(RoundParameters parameters, WasabiRandom random, int minInputCountBlameRound)
```

Comment:
Why do we need to store this "constant" from the Config inside the `Round` object?

Wouldn't it be easier to extract the `AreEnoughInputsForBlameRound` to `Arena` or somewhere and simply call `AreEnoughInputsForBlameRound(round.InputCount, Config.MinInputCountByBlameRound)`

Reply:
Ah ofc. 
Sorry for running the same rounds over and over again.